doc/results.Rmd

```diff
@@ -255,11 +255,15 @@ A convention worth following is to make each table a sub section. This makes the
 
 ```{r}
 
-footer_tbl_val <- "Table values are n (%) or median (25th, 75th percentile)"
+footer_tbl_val <- "Table values are n (%) or mean (SD)"
 
 data %>% 
   infuse_dictionary(meta, units = 'descriptive') %>% 
-  tbl_summary(by = 'island', missing = 'no') %>% 
+  tbl_summary(by = 'island', missing = 'no',
+  statistic = list(
+    all_continuous() ~ "{mean} ({sd})",  # Mean and SD for continuous
+    all_categorical() ~ "{n} ({p}%)"  # Custom format for categorical
+  )) %>% 
```

Comment:
Nice idea! Great use of the `statistic` input and the code is very readable. This is a fantastic contribution🥇